windows/jitsi-meet/index.html

```diff
@@ -0,0 +1,15 @@
+ <!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="../../css/welcome.css">
+        <script src="https://meet.jit.si/external_api.js"></script>
+        <script>
+            require("./render.js");
```

Comment:
But there is already main.js file exist in the root directory. I put render.js as the name because it runs in the render process of Electron. Maybe change name to app.js instead?